src/Test.First/SumEvens.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace Test.First
+{
+    public class SumEvens
+    {
+        public int Sum(int[] arr)
+        {
+            var sum = 0;
+            foreach (var number in arr)

```

Comment:
There is a cool extension to the c# language called linq that makes code ever more expressive.

Can you rewrite you re solution using linq?